anonlink/similarities/_dice_x86.py

```diff
@@ -111,4 +112,20 @@ def dice_coefficient_accelerated(
         result_indices0.extend(repeat(i, matches))
         result_indices1.extend(c_indices[0:matches])
 
+    np_sims = np.frombuffer(result_sims, dtype=result_sims.typecode)
+    np_indices0 = np.frombuffer(result_indices0,
+                                dtype=result_indices0.typecode)
+    np_indices1 = np.frombuffer(result_indices1,
+                                dtype=result_indices1.typecode)
+    
+    np.negative(np_sims, out=np_sims)  # Sort in reverse.
+    # Mergesort is stable. We need that for correct tiebreaking.
+    order = np.argsort(np_sims, kind='mergesort')
+    np.negative(np_sims, out=np_sims)
+
+    # This modifies the original arrays since they share a buffer.
+    np_sims[:] = np_sims[order]
+    np_indices0[:] = np_indices0[order]
+    np_indices1[:] = np_indices1[order]
```

Comment:
This is a bit of dejavu from the previous code section. However I assume you want to leave it separate so we can more easily optimize.

Reply:
Hmmm, maybe it'd be better to put it in a separate function for now…